src/ppx/grfx/grfx_swapchain.cpp

```diff
@@ -312,6 +395,18 @@ grfx::RenderPassPtr Swapchain::GetRenderPass(uint32_t imageIndex, grfx::Attachme
     return object;
 }
 
+grfx::RenderTargetViewPtr Swapchain::GetRenderTargetView(uint32_t imageIndex, grfx::AttachmentLoadOp loadOp) const
+{
+    grfx::RenderTargetViewPtr object;
+    GetRenderTargetView(imageIndex, loadOp, &object);
+    return object;
+}
+
+grfx::DepthStencilViewPtr Swapchain::GetDepthStencilView(uint32_t imageIndex) const
+{
```

Comment:
Should this function range check imageIndex as well?

Reply:
range checking is now handled by the other function GetDepthStencilView